external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py

```diff
@@ -133,3 +137,70 @@ def class_accuracy(self, results, gt_labels):
             cls_acc = np.sum(cls_pred) / len(cls_pred)
             accracies.append(cls_acc)
         return accracies
+
+
+@DATASETS.register_module()
+class MPAMultilabelClsDataset(MPAClsDataset):
+    def load_annotations(self):
+        for dataset_item in self.ote_dataset:
+            if dataset_item.get_annotations() == []:
+                label = None
+            else:
+                label = int(dataset_item.get_annotations()[0].get_labels()[0].id_)
```

Comment:
It would be better to map using self.label_idx like you did for ignored_labels

Reply:
Updated with more clear code. Thank you for suggestion.